### PR TITLE
📝 Add feature request process

### PR DESCRIPTION
**Pull request by bagg3rs**
# Pull Request Objective

This adds the Feature Request google doc [here](https://docs.google.com/document/d/1YoRyf7AOcMtO_6ccX7BrzbLISXb4YN9TgilsDuqEhuo/edit) into our docs
@Ed-Bajo FYI

### CODE REVIEW

File: source/documentation/platform/team-processes/feature-requests.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+- Issue gets refined by the team.
+- Once the requests are prioritised, the decision to bring that into the sprint is made.
+- The agreed plan is to use the original feature request ticket to deal with the request. Stories may be created by the team to create a new ticket where the story would be defined, making sure there is a definition of done etc, with additional issues linked to the feature request.
+
+![Diagram](images/feature-request-flow.excalidraw.png)
+
+## Amigos Team
+
+- Edwin Bajomo - Product Manager
```
Comment (michaeljcollinsuk) [inline source/documentation/platform/team-processes/feature-requests.html.md.erb]: Given today is Edwins last day with AP, is it worth updating to Darren?

Comment (michaeljcollinsuk) [reply source/documentation/platform/team-processes/feature-requests.html.md.erb]: Or perhaps all those changes will happen together when Darren starts officially

**Review by michaeljcollinsuk — DISMISSED** [review]

File: source/documentation/platform/team-processes/feature-requests.html.md.erb (hunk 1 above)
Comment (bagg3rs) [reply source/documentation/platform/team-processes/feature-requests.html.md.erb]: Is this table list useful?

Comment (michaeljcollinsuk) [reply source/documentation/platform/team-processes/feature-requests.html.md.erb]: Depends if the Amigos team is listed elsewhere - only needs to be in one place really

File: source/documentation/platform/team-processes/feature-requests.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+# <%= current_page.data.title %>
+
+This document will explain how we respond to [feature requests](https://github.com/ministryofjustice/analytical-platform/issues/new?assignees=&labels=feature-request&projects=&template=feature-request-template.yml&title=+%E2%9C%A8+%3Ctitle%3E) that are created by our users and our team.
+
+## Requests to the team
+
+Requests can come in a few different paths the most common are;
+
+- Analytical Platform user raises [feature request](https://github.com/ministryofjustice/data-platform-support/issues/new/choose)
+- Discussion with user via email, slack where our team will ask if they can raise a formal request or do so on their behalf if for example they do not have an Analytical Platform account
```
Comment (jacobwoffenden) [inline source/documentation/platform/team-processes/feature-requests.html.md.erb]:
```suggestion
- Discussion with user via email or Slack where our team will ask if they can raise a formal request or do so on their behalf if for example they do not have a GitHub account
```

File: source/documentation/platform/team-processes/feature-requests.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+- Team members can raise [requests](https://github.com/ministryofjustice/analytical-platform/issues/new?assignees=&labels=feature-request&projects=&template=feature-request-template.yml&title=+%E2%9C%A8+%3Ctitle%3E) based on a [support issue](https://github.com/ministryofjustice/data-platform-support/issues) or engagement with users
+
+## Amigos reviews
+
+The Amigo's will review the [feature-request](https://github.com/ministryofjustice/analytical-platform/labels/feature-request) labelled issues and establish if there is enough information. If not, the Product Manager or Team will contact the requestor and gather more information
```
Comment (jacobwoffenden) [inline source/documentation/platform/team-processes/feature-requests.html.md.erb]:
```suggestion
The Amigos will review the [feature-request](https://github.com/ministryofjustice/analytical-platform/labels/feature-request) labelled issues and establish if there is enough information. If not, the Product Manager or Team will contact the requestor and gather more information
```

File: source/documentation/platform/team-processes/feature-requests.html.md.erb [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+## Prioritisation of the Feature Request
+
+- Once the necessary information is captured, the Amigos prioritised the requests based on the following
```
Comment (jacobwoffenden) [inline source/documentation/platform/team-processes/feature-requests.html.md.erb]:
```suggestion
- Once the necessary information is captured, the Amigos prioritise the requests based on the following
```

File: source/documentation/platform/team-processes/feature-requests.html.md.erb [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+- Criticality of request to the users of the Analytical Platform performing their duties
+Value the request will add to user experience of the platform
+Timelines of the requests (when it needs to be done)
```
Comment (jacobwoffenden) [inline source/documentation/platform/team-processes/feature-requests.html.md.erb]:
```suggestion
  - Criticality of request to the users of the Analytical Platform performing their duties
  - Value the request will add to user experience of the platform
  - Timelines of the requests (when it needs to be done)
```

File: source/documentation/platform/team-processes/feature-requests.html.md.erb [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+## Feature Request brought into the sprint
+
+- Issue gets refined by the team.
+- Once the requests are prioritised, the decision to bring that into the sprint is made.
+- The agreed plan is to use the original feature request ticket to deal with the request. Stories may be created by the team to create a new ticket where the story would be defined, making sure there is a definition of done etc, with additional issues linked to the feature request.
```
Comment (jacobwoffenden) [inline source/documentation/platform/team-processes/feature-requests.html.md.erb]: minor nit - do we use full stops?
```suggestion
- Issue gets refined by the team
- Once the requests are prioritised, the decision to bring that into the sprint is made
- The agreed plan is to use the original feature request ticket to deal with the request. Stories may be created by the team to create a new ticket where the story would be defined, making sure there is a definition of done etc, with additional issues linked to the feature request
```

**Review by jacobwoffenden — CHANGES_REQUESTED** [review]

File: source/documentation/platform/team-processes/feature-requests.html.md.erb (hunk 6 above)
Comment (bagg3rs) [reply source/documentation/platform/team-processes/feature-requests.html.md.erb]: not on lists

**Review by jacobwoffenden — APPROVED** [review]
LGTM

**Pull request merged**